src/devices/cpu/z80/lz8420m.cpp

```diff
@@ -0,0 +1,47 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
```

Comment:
That's not your name :P 
Also in the .h file

Reply:
That is a copy-and-paste issue, I will fix that.